content/docs/setup/kubernetes/upgrading-istio/index.md

```diff
@@ -255,3 +255,61 @@ spec:
     tls:
       mode: DISABLE
 {{< /text >}}
+
+## Migrating the `RbacConfig` to `ClusterRbacConfig`
+
+The `RbacConfig` has a [bug](https://github.com/istio/istio/issues/8825). The bug reduces the scope of
+the object to be namespace-scoped in some cases. The `ClusterRbacConfig` follows the exact same
+specification as the `RbacConfig` but with correct cluster scope implementation.
+
+1. Replace the custom resource of kind `RbacConfig` with kind `ClusterRbacConfig` if you use
+   [Istio authorization](/docs/concepts/security/#authorization) with custom resource of kind `RbacConfig`:
+
+    A script is provided to automate the migration for you. Depending on if you have the [Istio installation package](/docs/setup/kubernetes/download-release):
+
+    - If you have the Istio installation package
+
+        Move to the Istio package directory and run the script with the following command:
+
+        {{< text bash >}}
+        $ ./tools/convert_RbacConfig_to_ClusterRbacConfig.sh
+        {{< /text >}}
+
+    - If you don't have the Istio installation package
+
+        Download and run the script directly with the following command:
+
+        {{< text bash >}}
+        $ curl -L https://raw.githubusercontent.com/istio/istio/master/tools/convert_RbacConfig_to_ClusterRbacConfig.sh | sh -
+        {{< /text >}}
+
+The script will get existing `RbacConfig` and apply a `ClusterRbacConfig` with same specification as
+Kubernetes doesn't allow to change the kind of a custom resource once it's created.
+
+For example, if you have a `RbacConfig` like the following one:
```

Comment:
The verification content starting here until the end must be rewritten as part of the procedure:
1. To verify the migration was completed successfully, take a RBAC configuration file before migration:

Reply:
What do you mean for verification content? This is not for verification, it just explains what would the script do.

Reply:
Readers expect to be able to verify what the script does after running it. It can be as simple as opening a file and verify the value was changed but there must be a verification step.